cmd/account/mgmt/account-unassign_test.go

```diff
@@ -21,8 +21,7 @@ import (
 )
 
 func TestAssumeRoleForAccount(t *testing.T) {
-	viper.Set(awsInternal.ProxyConfigKey, "")
-	viper.Set(awsInternal.SkipProxyCheckKey, true)
+	viper.Set(awsInternal.ProxyConfigKey, "a-random-proxy")
```

Comment:
Is this left over from testing? Either way I think we should default to empty-string still.

Reply:
_sigh_ That's my bad. I totally missed that this was the test file. Carry on.